src/GraphQL/Execution/SubscriptionExecutionStrategy.cs

```diff
@@ -35,15 +35,15 @@ public SubscriptionExecutionStrategy(IExecutionStrategy baseExecutionStrategy)
     /// <summary>
     /// Executes a GraphQL subscription request and returns the result. The result consists
     /// of one or more streams of GraphQL responses, returned within <see cref="SubscriptionExecutionResult.Streams"/>.
-    /// No serializable <see cref="ExecutionResult"/> is directly returned unless an error has occurred. This relates more to the protocol in use (defined in the transport layer) than the response here.
+    /// No serializable <see cref="ExecutionResult"/> is directly returned unless an error has occurred. This relates
+    /// more to the protocol in use (defined in the transport layer) than the response here.
     /// <br/><br/>
     /// Keep in mind that if a scoped context is passed into <see cref="ExecutionContext.RequestServices"/>,
     /// and if it is disposed after the initial execution, node executions of subsequent data events will contain
     /// the disposed <see cref="ExecutionContext.RequestServices"/> instance and hence be unusable.
     /// <br/><br/>
-    /// If scoped services are needed, it is recommended to override
-    /// <see cref="ProcessDataAsync(ExecutionContext, ExecutionNode, object?)"/>
-    /// with an implementation that creates a service scope during execution of the data events.
+    /// If scoped services are needed, it is recommended to utilize the ScopedSubscriptionExecutionStrategy
+    /// class from the GraphQL.MicrosoftDI package, which will create a service scope during execution of data events.
```

Comment:
```suggestion
    /// class from the GraphQL.MicrosoftDI package, which will create a service scope during processing of data events.
```

I think that GraphQL requests are _executed_ but data events are _processed_.